src/ensembl/src/content/app/App.tsx

```diff
@@ -64,6 +65,10 @@ const AppInner = (props: AppProps) => {
         <Route path={`${url}/global-search`} component={GlobalSearch} />
         <Route path={`${url}/species-selector`} component={SpeciesSelector} />
         <Route path={`${url}/custom-download`} component={CustomDownload} />
+        <Route
+          path={`${url}/entity-viewer/:genomeId?/:entityId?`}
```

Comment:
Nothing forcing us to be consistent.
It is rumoured to be somewhat better SEO-wise to keep important bits of your url in the path rather than in the query params. Entity viewer is the application that will have to be indexable by search engines; genome browser — probably not.